experimental/dds/tree2/src/test/snapshots/summary.spec.ts

```diff
@@ -58,3 +71,74 @@ describe("snapshot tests", () => {
 		});
 	}
 });
+
+const schemaTrees: {
+	only?: boolean;
+	skip?: boolean;
```

Comment:
addition of this test suite to this folder means we should update the containing README. I'd add some sections / reorganize existing sections to make them clear which portion of the tests they talk about.

It might make it easier to do such a rewrite if you put these snapshot tests into a different file. Then you'll also need to update the `test:snapshot:regen` command in package.json, but that shouldn't be too bad (I think 'all test files in the folder experimental/dds/tree2/src/test/snapshots' is reasonable enough)

Reply:
Updated readme, but wasn't sure if the changes were what you were imagining.